docs/DeveloperGuide.md

```diff
@@ -451,52 +463,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 2.
 
 
-**Use case: Deleting a single resident**
+**Use case: UC 7. Deleting a single resident**
 
 **MSS**
-1. Resident moves out of RC4.
-2. User uses the delete command to remove the resident from RC4HDB.
+1. Resident moves out of [**RC4**](#glossary).
+2. User deletes the resident.
 3. RC4HDB removes the corresponding resident from the database.
 4. RC4HDB displays the details of that resident that has been deleted.
 
     Use case ends.
 
-**Extension**
+**Extensions**
 
-* 2a. User enters an invalid index.
+* 2a. User enters an invalid input.
 
     * 2a1. RC4HDB shows an error message.
 
       Use case resumes at step 2.
 
 
-* 3a. RC4HDB cannot find any resident matching the user input.
+* 2b. RC4HDB cannot find any resident matching the user input.
 
-    * 3a1. RC4HDB shows an error message.
+    * 2b1. RC4HDB shows an error message.
 
       Use case resumes at step 2.
 
 
-**Use case: Clearing all data**
+**Use case: UC 8. Clearing all data**
 
 **MSS**
 1. User wants to clear all data from the current working file.
-2. User uses the clear command.
-3. RC4HDB replaces the current data file with an empty data file with the same name.
-4. RC4HDB shows a success message.
+2. RC4HDB replaces the current data file with an empty data file with the same name.
+3. RC4HDB shows a success message.
```

Comment:
Perhaps the 2nd step might show too much of the implementation.
We can simply change it to "**RC4HDB clears all data from the current working file.**"